src/components/duelling-picklist/duelling-picklist.style.js

```diff
@@ -30,6 +35,53 @@ const StyledDuellingPicklist = styled.div`
   ${StyledPicklist}:last-of-type {
     padding-left: 36px;
   }
+  ${StyledPicklist}:only-of-type {
+    padding-left: 4px;
+  }
+  ${StyledDraggableContainer} {
+    width: 100%;
+    height: 100%;
+    overflow-x: visible;
+    margin: 0;
+    padding-right: 0;
+    ${({ theme }) =>
+      css`
+        scrollbar-color: ${theme.scrollbar.light.thumb}
+          ${theme.scrollbar.light.track};
+
+        &::-webkit-scrollbar {
+          width: 8px;
+        }
+
+        &::-webkit-scrollbar-thumb {
+          background-color: ${theme.scrollbar.light.thumb};
+        }
+
+        &::-webkit-scrollbar-track {
+          background-color: ${theme.scrollbar.light.track};
+        }
+      `}
+
+    ${StyledIcon}:last-of-type {
+      display: none;
+    }
+    ${StyledPicklistItem} {
```

Comment:
One more thing, in scenario of a DuellingPicklist which already has some items selected, this will result in the animation being run on all the items as shown below:

https://user-images.githubusercontent.com/20651022/133985647-8b25e92a-82e2-4a0b-903e-c27e3d1407eb.mov